crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -1211,27 +1227,39 @@ pub fn parse_conditions<V: SpendVisitor>(
                 }
             }
             Condition::CreateCoinAnnouncement(msg) => {
-                decrement(&mut announce_countdown, msg)?;
+                if (flags & COST_CONDITIONS) == 0 {
+                    decrement(&mut announce_countdown, msg)?;
+                };
                 state.announce_coin.insert((spend.coin_id.clone(), msg));
             }
             Condition::CreatePuzzleAnnouncement(msg) => {
-                decrement(&mut announce_countdown, msg)?;
+                if flags & COST_CONDITIONS != COST_CONDITIONS {
```

Comment:
```suggestion
                if flags & COST_CONDITIONS == 0 {
```
Perhaps this is a matter of taste. I prefer this one, and right now it's not consistent, which looks a bit odd.

Reply:
standardised